client/me/purchases/cancel-purchase/test/cancellation-effect.jsx

```diff
@@ -69,8 +69,7 @@ describe( 'cancellation-effect', () => {
 				productsValues.isGSuiteOrGoogleWorkspace = () => true;
 				const headline = cancellationEffectDetail( purchase, translate );
 				expect( headline.text ).to.equal(
-					'You will be refunded %(cost)s, but your G Suite account will continue working without interruption. ' +
-						'You will be able to manage your G Suite billing directly through Google.'
+					'You will be refunded %(cost)s, and your %(googleMailService)s account will continue working without interruption. You will be able to set up billing for that account directly with Google.'
```

Comment:
As above, this may be simpler as "your account" instead of "that account" in the second sentence.